src/session/src/context.rs

```diff
@@ -32,46 +34,55 @@ impl Default for QueryContext {
     }
 }
 
+impl Display for QueryContext {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "QueryContext{{catalog: {}, schema: {}}}",
+            self.current_catalog(),
+            self.current_schema()
+        )
+    }
+}
+
 impl QueryContext {
     pub fn arc() -> QueryContextRef {
         Arc::new(QueryContext::new())
     }
 
     pub fn new() -> Self {
         Self {
-            current_catalog: ArcSwapOption::new(None),
-            current_schema: ArcSwapOption::new(None),
+            current_catalog: ArcSwap::new(Arc::new(DEFAULT_CATALOG_NAME.to_string())),
+            current_schema: ArcSwap::new(Arc::new(DEFAULT_SCHEMA_NAME.to_string())),
         }
     }
 
-    pub fn with(catalog: String, schema: String) -> Self {
+    pub fn with(catalog: &str, schema: &str) -> Self {
         Self {
-            current_catalog: ArcSwapOption::new(Some(Arc::new(catalog))),
-            current_schema: ArcSwapOption::new(Some(Arc::new(schema))),
+            current_catalog: ArcSwap::new(Arc::new(catalog.to_string())),
+            current_schema: ArcSwap::new(Arc::new(schema.to_string())),
         }
     }
 
-    pub fn current_schema(&self) -> Option<String> {
-        self.current_schema.load().as_deref().cloned()
+    pub fn current_schema(&self) -> String {
```

Comment:
Why not just return a &str?

Reply:
`current_schema` is guarded by ArcSwap, hard to return a ref into it